### PR TITLE
Added like feature

### PR DESCRIPTION
**Pull request by Hakdoff**
OVERVIEW
Added like and unlike feature

NOTES
make migrate

### CODE REVIEW

File: src/app/Services/PostService.php [hunk 1]
```diff
@@ -55,9 +55,26 @@ public function deletePost(Post $post): Void
...
         return auth()->user()->id === $post['user_id'] && $post->update($request->all());
     }
+    
+    /**
+     * Like Post
+     */
+    public function like(Post $post): void
+    {
+        $liker = auth()->user();      
+        $liker->likes()->attach($post);
```
Comment (mikco-hipe) [inline src/app/Services/PostService.php]:
```suggestion 
        auth()->user()->likes()->attach($post);

File: src/app/Services/PostService.php [hunk 2]
```diff
@@ -55,9 +55,26 @@ public function deletePost(Post $post): Void
...
+    }
+    
+    /**
+     * Unlike Post
+     */
+    public function unlike(Post $post): void
+    {
+        $liker = auth()->user();
+        $liker->likes()->detach($post);
```
Comment (mikco-hipe) [inline src/app/Services/PostService.php]:
```suggestion 
        auth()->user()->likes()->detach($post);

Comment (Hakdoff) [reply src/app/Services/PostService.php]: Make some changes Sir

File: src/app/Services/PostService.php [hunk 3]
```diff
@@ -55,9 +55,26 @@ public function deletePost(Post $post): Void
...
+        auth()->user()->likes()->attach($post);
+    }
+    
+    /**
+     * Unlike Post
+     */
+    public function unlike(Post $post): void
+    {
+        // $liker = auth()->user();
+        // $liker->likes()->detach($post);
```
Comment (mikco-hipe) [inline src/app/Services/PostService.php]: may naiwan

Comment (Hakdoff) [reply src/app/Services/PostService.php]: Removed unnecessary comments Sir

**Review by mikco-hipe — APPROVED** [review]
LGTM 🙇

**Pull request merged**